commands/core/core.drush.inc

```diff
@@ -499,7 +499,7 @@ function _core_site_credentials($right_margin = 0) {
 function _core_path_aliases($project = '') {
   $paths = array();
   $site_wide = drush_drupal_sitewide_directory();
-  $boot = drush_get_bootstrap_object();
+  $boot = \Drush::getBootstrap();
```

Comment:
In D8, this method would usually be called Bootstrap, and not getBootstrap. See the names in Drupal.php ... I'm OK with or without the 'get'.